### PR TITLE
Position of error message

### PR DESCRIPTION
**Pull request by shreya-fyle1**

### CODE REVIEW

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 1]
```diff
@@ -624,11 +624,11 @@
   &--amount-error {
     color: $red;
     font-size: 12px;
-    margin: -8px 0 0 33%;
+    margin: -8px 0 0 81.575px;
```
Comment (yksolanki9) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:627]: Specifying the exact value wont work. This would break for different screen sizes

**Review by yksolanki9 — CHANGES_REQUESTED** [review]

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 2]
```diff
@@ -624,11 +624,11 @@
...
   }
 
   &--orig-amount-error {
-    margin: -68px 4px 48px 52%;
+    margin: -68px 4px 48px 81.575px;
```
Comment (yksolanki9) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:631]: Same as above

**Review by yksolanki9 — CHANGES_REQUESTED** [review]

**Review by yksolanki9 — APPROVED** [review]
Test it for different screen sizes and with and without foreign currency before merging

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 3]
```diff
@@ -628,7 +628,7 @@
   }
 
   &--orig-amount-error {
-    margin: -68px 4px 48px 52%;
+    margin: -68px 4px 48px 33%;
```
Comment (Dimple16) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:631]: check on different screen resolutions, this might break in a few devices

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss (hunk 3 above)
Comment (shreya-fyle1) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.scss:631]: I checked every resolutions and dimensions and each of them is working well and none of them is breaking. So we can go with this PR.

**Review by Dimple16 — CHANGES_REQUESTED** [review]
please add screenshots of the change

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss (hunk 3 above)
Comment (shreya-fyle1) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.scss:631]: Screenshot_2022-10-27-19-03-13-50_cb3351a6e9832df473867ce5e08766b8
Before fixing

**Comment by shreya-fyle1** [comment]
Screenshot (511)
After fixing

**Comment by shreya-fyle1** [comment]
Screenshot (512)
Without foreign currency

**Comment by shreya-fyle1** [comment]
Screenshot (513)
With foreign currency

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**